config/experimental/s3configsource/config.go

```diff
@@ -0,0 +1,19 @@
+package s3configsource
```

Comment:
Missing licence header.

Reply:
Maybe rename this to awss3configsource, it's more mouthful but we also prefix the AWS/Amazon components with "aws".